internal/liqonet/tunnelEndpointCreator/tunnelEndpointCreator-config.go

```diff
@@ -12,39 +16,36 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
 	"k8s.io/klog"
-	"net"
-	"os"
-	"reflect"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func (tec *TunnelEndpointCreator) GetConfiguration(config *configv1alpha1.ClusterConfig) (map[string]*net.IPNet, error) {
+func (tec *TunnelEndpointCreator) GetConfiguration(config *configv1alpha1.ClusterConfig) (map[string]string, error) {
 	correctlyParsed := true
-	reservedSubnets := make(map[string]*net.IPNet)
+	reservedSubnets := make(map[string]string)
```

Comment:
I've thought the same but in func UpdateConfiguration we need maps in order to easily understand if all networks in the tep exist in the set of reserved network taken by clusterconfig.
Otherwise, we have to use two nester for loop